requirements.txt

```diff
@@ -1,9 +1,13 @@
 asyncpg
-psycopg2
+psycopg2-binary
```

Comment:
Self Review:

I always use docker on this machine so I don't have the headers of postgres installed to allow me to install `psycopg2` without the `binary`.